content/posts/welcome-to-our-beta-github-app.md

```diff
@@ -0,0 +1,72 @@
+title: Welcome to our GitHub App (beta access)
+date: 2025-06-02
+description:
+category: Feature announcement
+tags: website, migration, github, authentication
+authors: Santos Gallegos
+status: published
+image: /images/github-app.png
+image_credit: Photo by <a href="https://unsplash.com/@synkevych?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Roman Synkevych</a> on <a href="https://unsplash.com/photos/black-and-white-penguin-toy-wX2L8L-fGeA?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+We are excited to announce that we are migrating from our legacy GitHub OAuth2 application to a modern [GitHub App](https://docs.github.com/en/apps/overview).
+You can start using the new GitHub App today by signing up for [beta access](#beta-access).
+
+## Why the change?
+
+GitHub Apps offer several improvements over traditional OAuth2 applications:
+
+- **More granular permissions.**
+  The GitHub App allows us to request only the permissions we need
+  -- no more broad, account-wide access.
+- **More control over repository access.**
+  You can choose exactly which repositories Read the Docs can access,
+  instead of granting access to all of them just to import a single one.
+- **No webhooks required.**
+  The GitHub App subscribes to the events it needs automatically when you install it.
+- **No need for deploy keys.**
+  To access private repositories the GitHub App uses a temporary read-only scoped token.
+- **Keeps working, even if access changes.**
+  If the user who originally connected the repository loses access,
+  the integration will continue working, since the GitHub App itself maintains access.
+- **Always in sync.**
+  The GitHub App subscribes to all required events and will always keep your project up to date with your repository.
```

Comment:
I don't think this is a difference, since we already have this on OAuth app.

Reply:
When using the OAuth app we don't keep repos in sync, we manually do the sync when the user signs in, and we also rely on having a webhook on the repo.

Reply:
Good point!